index.html

```diff
@@ -59,6 +59,46 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        As the popularity of devices with immersive capabilities continues to
```

Comment:
This is still somewhat written in an aspirational manner... we should really frame it like it's 10 years in the future and it's stating facts (i.e., immersive capabilities are well established, people use this spec to embed 3d objects into web pages).